cves/kernel/CVE-2015-2041.yml

```diff
@@ -84,14 +86,9 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: 6b8d9117ccb4f81b1244aafa7bc70ef8fa45fc49
   note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    Manually confirmed
 vcc_instructions: |
   The vulnerability-contributing commits.
 
```

Comment:
I am giving this a 0 as the exploit reveals data already accessible by the user.